nttt/tidyup.py

```diff
@@ -126,6 +128,14 @@ def tidyup_translations(arguments):
 
             print("Complete")
```

Comment:
`print("Complete")` should probably go after the added piece of code

Reply:
Of course; moved